src/main/xar-resources/data/lucene/lucene.xml

```diff
@@ -314,27 +314,33 @@
                     These parameters must match a <code>Constructor</code> signature on the
                     underlying Java class of the Analyzer, please review the Javadoc for the
                     Analyzer that you wish to configure.</para>
-                <para>We currently support passing the following types: </para>
+                <para>We currently support passing the following types:</para>
                 <itemizedlist>
                     <listitem>
-                        <para> <code>String</code> (default if no type is specified)</para>
+                        <para><code>String</code> (default if no type is specified)</para>
                     </listitem>
                     <listitem>
-                        <para> <code>java.io.FileReader</code> (since Lucene 4) or <code>file</code>
+                        <para><code>String[]</code> (since eXist-db 5.4.0)</para>
+                    </listitem>
+                    <listitem>
+                        <para><code>char[]</code> (since eXist-db 5.4.0)</para>
```

Comment:
The type should be changed to `java.lang,char[]` as that is what needs to be used.

Reply:
It should be `char[]`